src/com/clover/remote/client/DefaultCloverConnectorListener.ts

```diff
@@ -1,136 +0,0 @@
-import sdk = require('remote-pay-cloud-api');
```

Comment:
This file is intended to be a template for developers...are we sure we want to remove this?

Reply:
👍 works for me as long as @mike-clover-com is ok with it.